gateway/api/views.py

```diff
@@ -201,7 +201,9 @@ def run_existing(self, request):
             carrier = {}
             TraceContextTextMapPropagator().inject(carrier)
             arguments = serializer.data.get("arguments")
-            token = request.auth.token.decode()
+            token = ""
+            if request.auth:
```

Comment:
Maybe it's needed I would need to take a look more in-depth but why are we adding now this `if`?

Reply:
I couldn't make the test to fake the `request.auth`.  It comes here without `auth` object here.  So I added this check.  I don't believe that this causes any bad effects in the runtime.

Reply:
Oh, I see it's due to the changes in the tests, then.